src/v3/src/components/PasswordRequirements/PasswordRequirements.tsx

```diff
@@ -101,18 +100,18 @@ const PasswordRequirements: UISchemaElementComponent<{
     <Box data-se="password-authenticator--rules">
       {/* @ts-ignore OKTA-471233 */}
       <Box marginBottom="s">
-        <Text as="span">{t('password.complexity.requirements.header')}</Text>
+        <Text as="span">{loc('password.complexity.requirements.header', 'login')}</Text>
```

Comment:
can this one be done in transformation layer?

Reply:
We would need to add a new field to the schema element specifically for this. If you are ok with that I can make that change.

Reply:
let's add a new field in schema, eventually I hope we can adopt the i18n[] approach